packages/realm-server/server.ts

```diff
@@ -365,7 +380,13 @@ export class RealmServer {
         },
       },
       {
-        ...(this.seedRealmURL ? { copiedFromRealm: this.seedRealmURL } : {}),
+        ...(this.seedRealmURL
+          ? {
+              copiedFromRealm: copyFromSeedRealm
+                ? this.seedRealmURL
+                : undefined,
+            }
+          : {}),
```

Comment:
it's easier to read if you just don't include it if `copyFromSeedRealm` is false (that's what this splat logic is doing)
```suggestion
        ...(this.seedRealmURL && copyFromSeedRealm
          ? {
              copiedFromRealm: this.seedRealmURL
            }
          : {}),
```